scripts/start-deployFTBA

```diff
@@ -45,7 +45,7 @@ if isTrue "$FTB_FORCE_REINSTALL" || ! [ -f "${ftbInstallMarker}" ] || [ "$(cat "
     log "Downloading FTB installer"
     if [ "$(uname -m)" == "aarch64" ]; then
       log "Downloading ARM installer"
-      $arm=/arm
+      $arm="/arm"
```

Comment:
And remove that leading $ I think

(Some languages include that there so now I'm getting myself confused 😀)

Reply:
yes because i was doing some php ealier this day :/